Dockerfile.compute-node

```diff
@@ -247,8 +247,8 @@ RUN wget https://github.com/pgvector/pgvector/archive/refs/tags/v0.7.0.tar.gz -O
     echo "1b5503a35c265408b6eb282621c5e1e75f7801afc04eecb950796cfee2e3d1d8 pgvector.tar.gz" | sha256sum --check && \
     mkdir pgvector-src && cd pgvector-src && tar xvzf ../pgvector.tar.gz --strip-components=1 -C . && \
     patch -p1 < /pgvector.patch && \
-    make -j $(getconf _NPROCESSORS_ONLN) PG_CONFIG=/usr/local/pgsql/bin/pg_config && \
-    make -j $(getconf _NPROCESSORS_ONLN) install PG_CONFIG=/usr/local/pgsql/bin/pg_config && \
+    make -j $(getconf _NPROCESSORS_ONLN) OPTFLAGS="" PG_CONFIG=/usr/local/pgsql/bin/pg_config && \
```

Comment:
A comment to explain why we pass `OPTFLAGS=""` would be nice. Something like:
```
# By default, pgvector Makefile uses `-march=native`. We don't want that, 
# because we build the images on different machines than where we run them.
# Pass OPTFLAGS="" to remove it.
```